### ISSUE TITLE
Error on main Javascript

### ISSUE THREAD

**Issue by Betrae** [jiahaog/nativefier#1001]
The converted web app is not opening and displaying an error box saying error on main javascript

**Comment by sarweshparajuli** [comment]
Can you specify your issue:
OS:?
Arch:?
Nativefier version:?
Nativefier command :?
Website:?

**Comment by slspeidel** [comment]
I am getting the same problem suddenly...perhaps after installing latest version?
![image](https://user-images.githubusercontent.com/4241311/86632178-4a3c1680-bf8c-11ea-83a7-1cde703c49a4.png)
My contact:  slspeidel@comcast.net

**Comment by nathanielban** [comment]
Adding a png icon to the indicated folder fixes the issue but indicates another issue in the build process.

**Comment by cmangla** [comment]
+1

| | |
|---:|---|
|OS| macOS Catalina 10.15.4|
|Arch| x86_64|
|Nativefier version| 9.0.0|
|Nativefier command| `nativefier --counter --name "Gmail" --internal-urls ".*?\.google\.*?" "gmail.com"`|
|Website| gmail.com|

I'm experiencing the same issue on MacOS. A workaround is to convert the already-generated icon to png using:
```
sips -s format png Gmail.app/Contents/Resources/electron.icns --out Gmail.app/Contents/Resources/app/icon.png
```

**Comment by Betrae** [comment]
> Can you specify your issue:
> OS:Windows 10
> Arch:x86_64
> Nativefier version:9.0.0
> Nativefier command :nativefier http://127.0.0.1:8000/ --name The project
> Website:local

**Comment by slspeidel** [comment]
Sorry, further specification of issue:
OS: OSX Catalina, Darwin-x64, 
Nativefier version: 8.2.3 , 
command: nativefier --single-instance --tray --bounce http://127.0.0.1:8080

**Comment by omarsotillo** [comment]
Using the version 8.0.7 seems to fix the issue. I had to lock the package.json to this version

IMHO `nativefier` should by default convert the images and the solution proposed here does not seem the goal solution.

**Comment by cmangla** [comment]
> solution proposed here does not seem the goal solution.

Certainly not; that's why it's been referred to as a _workaround_.

**Comment by ronjouch** [comment]
Closing as dup of https://github.com/jiahaog/nativefier/issues/992 , which should be fixed in f234473. Please try it with a [developer build](https://github.com/jiahaog/nativefier/blob/master/docs/development.md), or wait for the next release.
